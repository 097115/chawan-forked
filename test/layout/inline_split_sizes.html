<!DOCTYPE html>
before <span style="background-color: red; line-height: 2em"> t<br>
<div>carthago delenda</div>
<span style="padding-left: 5em; line-height: 2em"></span><span>est</span><br>
hello</span> after
