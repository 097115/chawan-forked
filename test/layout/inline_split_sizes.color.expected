       [48;2;255;0;0m [m
[38;2;255;255;255mbefore [48;2;255;0;0mt[m[38;2;255;255;255m[m

[38;2;255;255;255mcarthago delenda[m
[48;2;255;0;0m             [m
[48;2;255;0;0m          [38;2;255;255;255mest[m
[48;2;255;0;0m     [m
[38;2;255;255;255m[48;2;255;0;0mhello[m[38;2;255;255;255m after[m

